pkg/services/ngalert/api/tooling/api.json

```diff
@@ -338,6 +338,7 @@
    "type": "object"
   },
   "AlertRuleNotificationSettingsExport": {
+   "description": "Field name mismatches with Terraform provider schema are noted where applicable.",
```

Comment:
Fixing this in my PR, technically shouldn't be in the spec.

Reply:
Rebased and regenerated